bip-0054.md

```diff
@@ -70,11 +70,14 @@ A limit is set on the number of potentially executed signature operations in val
 transaction. It applies to all transactions in the block except the coinbase transaction[^1]. For
 each input in the transaction, count the number of `CHECKSIG` and `CHECKMULTISIG` in the input
 scriptSig and previous output's scriptPubKey, including the P2SH redeemScript. The accounting is the
-same as for [bip-0016][BIP16 specs]: a `CHECKSIG`/`CHECKSIGVERIFY` operation accounts for 1 and a
-`CHECKMULTISIG`/`CHECKMULTISIGVERIFY` accounts for the number of public keys associated, or 20 if
-the number of public keys is greater than 16. A `CHECKMULTISIG`/`CHECKMULTISIGVERIFY` not directly
-preceded by a minimally-pushed number between 1 and 16 (included) accounts for 20. If the
-total is strictly higher than 2500, the transaction is invalid.
+same as for [bip-0016][BIP16 specs], evaluating the scriptSig, scriptPubKey, and P2SH redeemScript
+separately:
+
+1. OP_CHECKSIG and OP_CHECKSIGVERIFY count as 1 signature operation, whether or not they are evaluated.
```

Comment:
```suggestion
1. `CHECKSIG` and `CHECKSIGVERIFY` count as 1 signature operation, whether or not they are evaluated.
```

(Here and below)

In the rest of the document opcodes are referred to without the "OP_" prefix and are `preformatted`.